dali/operators/reader/loader/loader.h

```diff
@@ -55,9 +55,11 @@ class Loader {
   using LoadTargetSharedPtr = std::shared_ptr<LoadTarget>;
   explicit Loader(const OpSpec& options)
     : shuffle_(options.GetArgument<bool>("random_shuffle")),
+      interleavement_(options.GetArgument<int>("interleavement")),
```

Comment:
```suggestion
      interleaving_(options.GetArgument<int>("interleaving")),
```
There's no such word: "interleavement".

Reply:
Thanks for pointing out the typo. I will move the data types to VideoLoader.

Reply:
Done. Renamed to interleave_size.